prac_09/taxi_simulator.py

```diff
@@ -0,0 +1,51 @@
+"""
+CP1404
+Taxi Simulator
+"""
+from taxi import Taxi
+from silver_service_taxi import SilverServiceTaxi
```

Comment:
have you tried using from prac_09.name_of_file? nevertheless, the import statement looks okay.

Reply:
To be honest I am not sure really what the difference is between having prac_09.filename and filename. But it would make sense in best practice to do it the first way, so that it is clear where the class is coming from.